lib/Physics/Unit.pm

```diff
@@ -1819,3 +1851,11 @@ Written by Chris Maloney <voldrani@gmail.com>
 Special thanks for major contributions and encouragement from Joel Berger.
 Thanks also to Ben Bullock, and initial help in formatting for distribution
 from Gene Boggs <cpan@ology.net>.
+
+=head1 COPYRIGHT AND LICENSE
+
+Copyright 2002-2003 by Chris Maloney
+
+This library is free software; you can redistribute it and/or modify
```

Comment:
This is incorrect. The license is now WTFPL, see #5 and https://github.com/Klortho/Physics-Unit/commit/2ba7c0ba87f6d038a7cd855643fdb73fdc7eb855

Reply:
I wanted to change the was contributors were mentioned anyway so I just went ahead and updated this.